resources/models/reservation.py

```diff
@@ -345,6 +345,10 @@ def get_notification_context(self, language_code, user=None, notification_type=N
                 'host_name': self.host_name,
                 'reserver_name': reserver_name,
                 'event_subject': self.event_subject,
+                'event_description': self.event_description,
+                'reserver_email_address': self.reserver_email_address,
```

Comment:
Can this be handled like `reserver_name`? If there is no `reserver_email_address` extra field set, use `user.email` if that is available.

Reply:
Yes, implement it as @jussih suggested